src/QGLView.cc

```diff
@@ -312,12 +312,13 @@ bool QGLView::save(const char *filename)
 
 void QGLView::wheelEvent(QWheelEvent *event)
 {
+  auto pos = event->pos();
```

Comment:
const

Reply:
Thanks for the review!  Sorry, didn't mean to ignore this so long, I went ahead and committed the added `const` to those functions.